notebooks/5094_harmonisation.py

```diff
@@ -169,22 +169,19 @@
         name="method",
     ).astype(str)
 
-    mask = pix.ismatch(
-        variable=[
-            "**Forest Burning**",
-        ]
-    )
-    user_overrides_gridding.loc[mask] = "reduce_offset_2030"
-
-    mask = pix.ismatch(
-        variable=[
-            "**CO|Energy Sector**",
-        ]
-    ) & user_overrides_gridding.index.get_level_values("region").str.contains(
-        "Eastern Africa|India|Rest of Southern Africa|Western Africa"
-    )
+    for idx, model_harm_year_value in model_pre_processed_for_gridding[
+        model_pre_processed_for_gridding[HARMONISATION_YEAR].index.get_level_values("variable").str.contains("Forest")
+    ][HARMONISATION_YEAR].items():
+        mask_history = (history_for_gridding_harmonisation.index.get_level_values("variable") == idx[3]) & (
+            history_for_gridding_harmonisation.index.get_level_values("region") == idx[2]
+        )
 
-    user_overrides_gridding.loc[mask] = "reduce_offset_2080"
+        if model_harm_year_value > 1.5 * history_for_gridding_harmonisation[mask_history][HARMONISATION_YEAR].item():
+            user_overrides_gridding[idx[0:3]] == "constant_ratio"
+        elif model_harm_year_value < 0.8 * history_for_gridding_harmonisation[mask_history][HARMONISATION_YEAR].item():
+            user_overrides_gridding[idx[0:3]] == "constant_offset"
+        else:
+            user_overrides_gridding[idx[0:3]] == "constant_offset_2030"
```

Comment:
```suggestion
            user_overrides_gridding[idx[0:3]] == "reduce_offset_2030"
```

? I don't think `constant_offset_2030` exists

Reply:
True! Sorry, my bad. This is what you meant @jkikstra, right?

I have updated it.

Reply:
Yep thanks for correcting!